modules/040-node-manager/images/bashible-apiserver/werf.inc.yaml

```diff
@@ -19,27 +19,20 @@ git:
 mount:
 - fromPath: ~/go-pkg-cache
   to: /go/pkg
-ansible:
+shell:
   beforeInstall:
-  - apk:
-      name: git
-      update_cache: yes
-  - command: rm -rf /var/cache/apk/*
+  - apk add git
   install:
-  - shell: go mod download
-    args:
-      chdir: /src
+  - cd /src
+  - go mod download
```

Comment:
I think better way export GOPROXY here

go mod download will use it

Reply:
Fixed.